### PR TITLE
add pry, pry-byebug, pry-remote to omnibus-chef

### PR DESCRIPTION
**Pull request by lamont-granquist**
removes pry as a dep of chef directly because it was never a dep of
chef.

### CODE REVIEW

**Comment by lamont-granquist** [comment]
@chef/client-core frequently requested, lets do it right.

File: omnibus/Gemfile.lock [hunk 1]
```diff
@@ -115,7 +115,7 @@ GEM
     nori (2.6.0)
     octokit (3.8.0)
       sawyer (~> 0.6.0, >= 0.5.3)
-    ohai (8.8.1)
+    ohai (8.9.0)
```
Comment (coderanger) [inline omnibus/Gemfile.lock]: Intended?

File: omnibus/config/projects/chef.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2012-2014 Chef Software, Inc.
+# Copyright 2012-2016 Chef Software, Inc.
```
Comment (coderanger) [inline omnibus/config/projects/chef.rb]: Can we pretty please do these as a single update rather than littering them in unrelated changes?

Comment (lamont-granquist) [inline omnibus/config/projects/chef.rb]: I would happily click merge on that PR.

File: omnibus/Gemfile.lock (hunk 1 above)
Comment (lamont-granquist) [inline omnibus/Gemfile.lock]: its necessary to pull master of omnibus-software and passes builds...

File: omnibus/config/projects/chef.rb (hunk 2 above)
Comment (coderanger) [inline omnibus/config/projects/chef.rb]: Done :)

Comment (lamont-granquist) [inline omnibus/config/projects/chef.rb]: lol, now i have merge conflicts....

File: omnibus/Gemfile.lock (hunk 1 above)
Comment (coderanger) [inline omnibus/Gemfile.lock]: :+1:

**Comment by coderanger** [comment]
:shipit:
